dev-games/box2d/box2d-2.3.1.recipe

```diff
@@ -35,6 +36,7 @@ BUILD_REQUIRES="
 	devel:libGLU$secondaryArchSuffix
 	"
 BUILD_PREREQUIRES="
+	cmd:ccache
```

Comment:
Really?

Reply:
Needs to go, but in the process of "rebuilding" it's nice to have around :)

Reply:
removed :)